04_distributed.ipynb

```diff
@@ -16,9 +16,7 @@
     "\t* [Making a cluster](#Making-a-cluster)\n",
     "\t\t* [Detailed method](#Detailed-method)\n",
     "\t\t* [Simple method](#Simple-method)\n",
-    "\t* [Executing with the distributed client](#Executing-with-the-distributed-client)\n",
-    "\t* [Distributed futures](#Distributed-futures)\n",
```

Comment:
"Executing with the distributed client" is really there, but the rest moved to notebook 06_distributed_advanced .